Simulator/Simulator.WebJob/app.config

```diff
@@ -2,19 +2,37 @@
 <configuration>
   <configSections>
     <sectionGroup name="applicationSettings" type="System.Configuration.ApplicationSettingsGroup, System, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b77a5c561934e089">
-      <section name="Microsoft.Azure.Devices.Applications.RemoteMonitoring.Simulator.WorkerRole.Properties.Settings" type="System.Configuration.ClientSettingsSection, System, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b77a5c561934e089" requirePermission="false" />
+      <section name="Microsoft.Azure.Devices.Applications.RemoteMonitoring.Simulator.WebJob.Properties.Settings" type="System.Configuration.ClientSettingsSection, System, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b77a5c561934e089" requirePermission="false" />
     </sectionGroup>
   </configSections>
-    <system.diagnostics>
-        <trace>
-            <listeners>
-                <add type="Microsoft.WindowsAzure.Diagnostics.DiagnosticMonitorTraceListener, Microsoft.WindowsAzure.Diagnostics, Version=2.7.0.0, Culture=neutral, PublicKeyToken=31bf3856ad364e35" name="AzureDiagnostics">
-                    <filter type="" />
-                </add>
-            </listeners>
-        </trace>
-    </system.diagnostics>
-<startup><supportedRuntime version="v4.0" sku=".NETFramework,Version=v4.5.1" /></startup>
+  <system.diagnostics>
+    <trace>
+      <listeners>
+        <add name="consoleListener" type="System.Diagnostics.ConsoleTraceListener" />
+        <add type="Microsoft.WindowsAzure.Diagnostics.DiagnosticMonitorTraceListener, Microsoft.WindowsAzure.Diagnostics, Version=2.7.0.0, Culture=neutral, PublicKeyToken=31bf3856ad364e35" name="AzureDiagnostics">
+          <filter type="" />
+        </add>
+      </listeners>
+    </trace>
+  </system.diagnostics>
+  <startup>
+    <supportedRuntime version="v4.0" sku=".NETFramework,Version=v4.5.1" />
+  </startup>
+  <appSettings>
+    <add key="docdb.DatabaseId" value="DevMgmtDB" />
+    <add key="docdb.DocumentCollectionId" value="DevMgmtCollection" />
+    <add key="DeviceRulesStoreContainerName" value="devicerules" />
+    <add key="DeviceRulesTableName" value="DeviceRulesNormalizedTable" />
+    <add key="ActionMappingStoreContainerName" value="actionmappings" />
+    <add key="ActionMappingStoreBlobName" value="mappings.json" />
+    <add key="AsaRefDataRulesBlobName" value="devicerules.json" />
+    <add key="SolutionName" value="Remote Monitoring Solution" />
+    <add key="TelemetryStoreContainerName" value="devicetelemetry" />
+    <add key="TelemetryDataPrefix" value="devicetelemetry/" />
+    <add key="TelemetrySummaryPrefix" value="devicetelemetry-summary/" />
+    <add key="AlertsStoreContainerName" value="rulesoutput" />
+    <add key="DeviceAlertsDataPrefix" value="" />
+  </appSettings>
```

Comment:
When I tried to run the Simulator.WebJob locally as a console app, it tried to load a setting with key `device.StorageConnectionString`, which doesn't exist here, and so it threw. Should that setting (and any others needed by the simulator) be included here?